lobby-db/src/main/resources/db/migration/V1.10.00.20190313.1200__delete_mute_by_username.sql

```diff
@@ -0,0 +1,21 @@
+
+-- mute only by network identifiers, drop mute by player name
+drop table muted_usernames;
+
+-- 'banned_usernames' will now keep track of player name black list with no expiration
```

Comment:
These 2 comments should probably reference username instead of player name.

Reply:
One comment in response 'user' is not allowed for a table name, so I was planning to have it be called the 'player_table'. So we'd have 'player_session' table. Thus at the DB level it would be called 'player'.

It's odd to have different names at different layers like that, so I would be in favor of transitioning to user 'player'.

Regardless, until the player table actually lands, I've no objection to update the 'player name' instances to 'username'.